### DISCUSSION TITLE
Name for Sunflower Land Citizens?

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1167] [Q&A] [answered]
The purpose of this discussion is to discuss a name that encapsulates all of the characters that live in the Sunflower MetaVerse.

The team is currently developing a range of character NFTs that will play a huge role in the game. The first iteration includes Humans and Goblins.

For game and marketing purposes we want to come up with a special name which represents these characters. This will be used throughout the game, Open Sea collections and social media.

**Characters**

To get an idea of the characters refer to the low fidelity sketches below:

![inprogress](https://user-images.githubusercontent.com/11745561/176982721-48854213-3833-41ba-add2-987e3180bf5d.jpeg)

**Theme**

We want to stick to a name which is friendly, casual and open for players of all types.

**Requirements**

A single name to represent our characters. This name will include all characters, all genders, all ethnicities and races (human, goblins, animals etc.)

**Examples**

- Cryptopians
- Sunflorians
- BlockBuddys

**Comment by Demonbane18** [comment]
Goblin Slayers (yes my pfp checks out lmao)

Serious suggestion though would be "Sunlites" for me dunno I'm not good with names

**Reply by adamhannigan** [reply]
I was waiting for this.... 🤣

**Reply by Demonbane18** [reply]
Hahaha

**Reply by eArL2384** [reply]
Ned Taters

Dan Bumpkin

Ana Sparsnip

Rose Moonflower 😝

**Reply by mdbabaylan** [reply]
Speedy Sunflower Farmers and The Patient Parsnippers

**Comment by Galphath** [comment]
Sunflower land, the land of Sun Farmers

**Comment by kharbush** [comment]
Farmingly  ,  Sulasun , flowerella ,i hope u like it

**Comment by kpin2** [comment]
Seedlings, Saplings, SunPotters, Ranch-Hands, SunRanchers, SunFarmers

**Reply by NotKeen** [reply]
I like SunFarmers a lot!

**Comment by sprmwhale5183** [comment]
Helianthians given that the scientific name for Sunflower is helianthus, it feels apt and not as simple as sunflorians :)

**Reply by sprmwhale5183** [reply]
Also relevant regardless of skills or profession, species of the chars because they all belong to Sunflower Land as Helianthians. Keeps it broad for future stuff that may revolve away from planting

**Comment by deathmbah666** [comment]
"rewolfnus" citizen..... is the best

**Comment by Chardyyy07** [comment]
Cultivator 🌻The Cultivatorist🌻 The Tillager

**Reply by AWatts13** [reply]
Not all characters can be assumed to be farmers and cultivators though, so it's not fully inclusive.

**Comment by kanxax** [comment]
Terrakians

**Comment by sfwhat** [comment]
My suggestions are below

Landlords
Landholders
Explorers
Farmkeepers
Crusadors
Voyagers
Cropsters
Farmlanders
Sunlanders
Florists

**Comment by IntegralDeFluxo** [comment]
Sunniers

**Reply by Kiulg** [reply]
That was the first I thought.

**Comment by o0j4yj4y0o** [comment]
What about calling them 'Sprouts'?

It's non-gendered, light, thematic and on brand?  Could also cover a variety of sprites that you choose to incorporate. =)

-Relwyn

Also, since it seems folks are kicking out some fun names I'll drop a couple from my homebrew D&D Game that I thought were entertaining:

Finnegan Brambleshoot
Octavious Scuttlebutt
Lumparms
Thane Copperstacks
Tyx
Uffizi
Yulon Bumblestump
Veldon Tenderborn
Wick

**Reply by AWatts13** [reply]
While on theme with being simple and cute, "Sprouts" feels a little too modern and farm/plant-centric to me. There's a little bit of a medieval theme within the game, and not all characters can be assumed to be farmers, I don't think. It also connotates that the characters are young / undeveloped. I don't think all players will appreciate that after working hard on leveling up and being bosses.

**Reply by o0j4yj4y0o** [reply]
Thanks for sharing your perspective!  I saw you mention in some of the other posts the need for simplicity and I tend to agree since to me, this is a branding discussion for their product.  Adam mentioned that the name would find it's way throughout the game, social media, open sea, etc. so, in my opinion, it's got to be something that's broad enough to encompass the world but targeted enough to catch attention.

In my head, Sprouts spoke a variety of disciplines and the idea that it would be something that a player would start new, care for and ultimately grow and develop.  Essentially a novice at a given discipline (Be it farming, baking, foraging, crafting, etc.)  which is something that everybody will start from and everybody can relate to.  I also thought it was broad enough that it could expand with the brand.  To your point on players not appreciating their leveling work being diminished, nothing prevents the broader grouping of 'Sprout' being augmented by something like, 'Master Sprout' upon reaching Level X or 'Elite Sprout' or 'Elder Sprout' after some condition is met.

I do enjoy Sunflorians if there's a broader narrative arc that the team is drawing from and they want incorporate lore about the Kingdom of Floria or some other idea.  I could also see the The Sprouts of Floria or The Sprouts of Sunfloria as well.  It's like the 'The Sims'.  

The name encompasses the brand. =)

**Comment by Elie11** [comment]
![abc](https://user-images.githubusercontent.com/40805682/176983720-839474b4-e5e5-42c2-a9b5-959d2c729cfe.JPG)

Zack (The Farmer)

Lily (The Gardener)

Son (The Harvester)

Gobta (The Shovel lover)

If needed for just one name a simple "The Farmer" or "The Horticulturist".

Hope you pick one from me XD

**Reply by AWatts13** [reply]
Is the inspiration for the name "Gobta" from the "That Time I Got Reincarnated as a Slime" anime? One of the main goblin characters is named Gobta.

**Comment by Fredfranz** [comment]
Epoy as the Farmer's name

**Comment by Genov29** [comment]
Hey man, it is hard to name characters if there isn't a lore behind it. Here is my Suggestion:

Assuming there is a kingdom, the Kingdom of Heliofloria (Sunfloria), it make sense if its citizens are called Sunfloreans, Floreans, Helians (reference to greek name of Sun God, Helios).

And my suggestion if there isn't a specific lore for citizens race name are Adarians (Taken from Adam's name), Suntevean (from Steve), Humereans (from Human) and Clyterians (from the myth of Clytie whom i believe watching helios as he and his chariots pulls the sun in the sky.)

**Reply by Genov29** [reply]
Oh and for the names of characters:

Humans: Taylor/Kyle (can be boy or girl)
Goblin: Gobta/Gobzo/Rigur/Gobro

**Reply by AWatts13** [reply]
Gobta and Rigur are from "That Time I Got Reincarnated as a Slime" - some anime fans here!

**Comment by Fredfranz** [comment]
I would suggest "INDAY" for the female gardener. I am basing my idea on a kind gardener next door which is we call her "Aling Inday".

**Comment by AKCH1N** [comment]
Name Ideas for The Farmer NFTs

- Organics (It represents anyone who leads an "organic" life, I think it's cute.)

- Nesters (Our farms is our "Nest" and these are the "Nesters" = means nest creators)

- Ranchers (More like "Farm Owners")

- Growers (People who feed and grow the seeds.)

- Harvesters (Other name of the "farmers" also represents a machine that useful in real farms.)

**Comment by pdjkxyz** [comment]
Suntizens, Sunnizens, Zenians are my suggestions

**Comment by JcEii** [comment]
**Islanders: The Mystic Adventurers**

Starring...

**Sterling** (The Adventurer)
![image](https://user-images.githubusercontent.com/105348036/176984069-3ade96bd-6e31-455e-bfd3-b460be230513.png)

**Alice** (The Archaeologist)
![image](https://user-images.githubusercontent.com/105348036/176984080-10c52f84-bab3-47bd-bbc5-ba909cd19942.png)

**Shaun** (The Farmer)
![image](https://user-images.githubusercontent.com/105348036/176984081-5c53b375-69b2-4653-a1ef-e1b4e42766cc.png)

**Austin** (The Seeker)
![image](https://user-images.githubusercontent.com/105348036/176984084-fae7285a-ff39-404d-9e71-d4428a56e736.png)


Also Starring...
Gabby (The Baker)
Ace (The Fisherman)
Mark (The quiet)
Cherry (The Crochetier)
Bella (The Journalist)
Abigail (The Biologist)
Juliet (The Banker)
Barry (The Mayor)
Gilbert (The Rich Guy)
Jeff (The Carpenter)
Lee (The Fighter)
Marshall (The Pirate)
Stephanie (The Bandit)

**Comment by stevew00dy** [comment]
Workfolk - they do our jobs for us 

rancher - typical name for a farmer

Agricultralists - Just because it sounds posh and upmarket

Sharecropper - a farmer especially in the southern U.S. who raises crops for the owner of a piece of land and is paid a portion of the money from the sale of the crops

- Yeoman 

- Sunflopians - Sounds fun

- Sunny Citizens - because I'm out of ideas

**Comment by stevew00dy** [comment]
As for names :D 

Farmer Giles, Lady Dirtpatch and The Almighty Greentoe

**Comment by AncientHorse** [comment]
Were nipples really necessary in base sketch?  This is supposed to be family friendly!  Apologies if it's 3D oriented and required from a design standpoint.  It's totally ok to chuckle if this wasn't the intent.

Edit: And as I look closer at male farmers, hahahahaha.

2nd Edit: Players should be able to chose any gender they want, including gender neutral.  Unpopular opinion, I think this is forcing players to identify with particular aspects of their body to identify with their character.  I think it should be more nebulous and players can chose the character they want to play without diagrams telling them what they should be.

**Reply by adamhannigan** [reply]
Players will be able to choose the initial gender + skin colour and visit a barber to receive a haircut.

The nipples definitely won't make the game 😅 . They are just included in prototype dimensions to help with overlay designs

**Comment by ShiningMinions** [comment]
"Grangers" = Slang term for farmers.

Others:
- Yokels
- Farmies
- Ranchers

Random names I thought of:
- Huckles (sounds like Huckleberry Finn)

**Comment by ZensouGaming** [comment]
just my opinion ..

SunFLanders.. (all farmers in the Sunflower Land)
![Farmer SFL](https://user-images.githubusercontent.com/108561941/176985338-72ab802a-838c-44a6-bd4b-6a72ccb1f2a9.png)

GobLanders.. (all goblins the village goblin)
![Goblin SFL](https://user-images.githubusercontent.com/108561941/176985345-88b65f38-af64-40a6-b1de-b5c155daeac1.png)

**Reply by totoyubar** [reply]
Tiny Goblin, Smart goblin, wisegoblin
Jose.
Joseph
Jackass
Piggy 
Pogy
Paggy

**Reply by JcEii** [reply]
I like this one '3'

**Comment by IvanSkazkin** [comment]
I liked your suggestion "**Sunflorians**"

**Comment by ImprobableHulk** [comment]
Some names I thought of:

- Yieldlings 
- Cultivites
- Seedlers

**Comment by gargula3004** [comment]
Farleysam

**Comment by Pimlka** [comment]
I second 'Sprouts' mentioned above, as I was going to suggest the same :)

Other suggestions for an overall name:
Sunnies
Bumpkins
Solians (sol = latin for sun)
Soilans/Soilians (soil)

**Reply by adamhannigan** [reply]
I have a soft spot for the word Bumpkin 😅 It's my Twitter handle!

**Reply by Pimlka** [reply]
Hah, nice. I've had a soft spot for it since playing the very old game Beasts and Bumpkins. Also, it is neutral in regards to race, gender etc. Could work!

**Comment by NotKeen** [comment]
Sunflorists / Sunflowerists
SunnyBuddies
SunPeople / SunnyPeople

**Comment by cripto-trader** [comment]
Why mot fuse SUNFLOWER LAND 
i suggest "Sulanians" - Su-nflower + Lan-d + nians..= Sulanians...hehe

**Reply by AWatts13** [reply]
breaking down the word using the "ian" suffix, it means "those from sulan." Sulan is of korean origin and means poached egg. hmmm.

**Comment by cripto-trader** [comment]
Again i suggest SULANIANS ,,just fused sunflower land

**Comment by cripto-trader** [comment]
Sulanians

**Comment by Omardvix** [comment]
Just to add my two cents here as well, I've thought of a few names. I just thought that the names should also reflect the characteristics of the farmers we want to be and not just a nice sounding name.

- **Greenkeepers:** They are individuals that dedicate their livelihood to keeping their land flourishing with life all year round, from flowers to vegetables and possibly even trees. 
- **Botanists:** They are experts in plants and take their time to study their behaviors like their growing time, crop yield, and mutations.
- **Sunnies:** These are people who brighten up the world around them, they bring life to the crops and animals that live on their far and bring great fortune to their home and community.

**Comment by GoblinNoLikePotato** [comment]
Potato, 
Potatoes.

**Comment by Blasta-alasta** [comment]
**Tanderran** taken from the Tanderra meaning 'a restful place' is one of the languages of the Indigenous Australians.

_“People of a restful place”… **The Tanderran**_

**Comment by cripto-trader** [comment]
𝗦𝘂𝗹𝗮𝗻𝗶𝗮𝗻𝘀 ( Su-nflower +Lan-d ) +nians

**Comment by cripto-trader** [comment]
Goodmorning Sulanians..😚

**Reply by Blasta-alasta** [reply]
Bro someone with the exact same name as you already suggested this!

**Reply by Yougottabekiddingme123** [reply]
lol

**Comment by Yougottabekiddingme123** [comment]
I think a good place to start would be to look at suffixes that could be used to come up with an all encompassing category for human farmers, goblins and other races that will be introduced later.

Suffix
(-ian) - One from, belonging to, relating to, or like something
(-ese) - Forms nouns indicating an inhabitant of a place.
(-oid) - Resembling, having the likeness, related to something

If we apply these using Adam's examples, we get:
Cryptopians - Cryptopese - Cryptoids
Sunflorians - Sunflorese - Sunfloroids/Sunfloids 

I personally like the -oids (Cryptoids, Sunfloroids, Sunfloids).
This can be broken down into Humanoids, Gobloids, etc

**Reply by sprmwhale5183** [reply]
Haemorrhoids?

**Comment by datorymen** [comment]
How about this?

Sunfloweree
Sunflowerees

**Comment by JustJayHSP** [comment]
Bumblebees, 
Pixelerians, 
Bumpkins, 
The Primes, 
The Origin, 
Budgrowers, 
Foragers, 
Argonomists
GoldenRods
Crofter
Herbalist

**Comment by ibmisibm** [comment]
Peasants

**Comment by vincentonecent** [comment]
sun - kin
kin as in one's family and relations.

**Comment by ThorBop** [comment]
My suggestion would be:
FlowerFolk

**Comment by Azerius** [comment]
► CornMan

Farm Animal Plants Universe
Fa ani pl uni
► Fanipluni xD

► Tractormer

► Sunsearcher

► Suncrafters

**Comment by johnnylcy** [comment]
“Old McDonald” had a sunflower farm

**Comment by Manor01G** [comment]
A few suggestions:

Thought Farmers
Earth's Guardians
The Children of hope 
The Guardians
Replenishers 
Good Earthlings

One Word Names:

Thoughtsies
Guardians 
Hopeies
Replenishers 
Avani - 
(Sanskrit origin) ancient Hindu name translating to earthly.
Dhara - 
(Sanskrit) meaning earth.
Edens - 
(Hebrew origin) means place of delight and inspired by the famous biblical garden of Eden.
Enkis - 
(Sumerian) is lord of the Earth in Sumerian mythology
Geo -
(Greek origin) meaning earth.
Yuri - 
(Russian) Meaning "earth-worker", this name is for nature-lovers.

**Comment by AWatts13** [comment]
For those coming up with some pretty fancy names, I think the simpler the better, as the whole point of this game is simplicity. Adam's suggestion of "Sunflorian" encapsulates that simplicity pretty well I think and is my top choice after thinking on it. 

Unlike some of the other suggestions, it doesn't drop to just "sun" which has a whole different feel and meaning from sunflower. "Sun" makes me think of strength, power, energy. Vs "sunflower" has a cutesy feel. The whole game has a cutesy feel. It also doesn't assume that all sunflower land people/creatures are farmers by using the word "Farm" or "Farmer." While it does shorten the word "sunflower" to "sunflor," it still semantically makes sense, in my opinion.

And as someone else said, the suffix "ian" means one from, belonging to, relating to, or like something. So "sunflor-ian" - someone from the land of sunflors.. sunflowers. It applies to all without excluding who you are or what you do and is true to the feel of the game: simple and cutesy.

"Good morning Sunflorians!"

Love it!

**Reply by AWatts13** [reply]
Oh besides being simple and cute, "sunflorian" also reminds me of a more medieval name for citizens, which is also on theme with the game, as opposed to some of the more modern sounding suggestions.

**Comment by ilNomade** [comment]
The seeds of the sunflower are its fruit and are called achenes. So for the citizens of sunflower land I propose "**Achenio**" in the singular and "**Achenes**" in the plural

[https://en.wikipedia.org/wiki/Achene](url)

**Comment by thisUsernameAlreadyExists** [comment]
`Agros` from [Agronomist](https://www.urbandictionary.com/define.php?term=Agronomist)
`Sunners` from [Sunner](https://www.urbandictionary.com/define.php?term=Sunner)

**Comment by mk-amorson** [comment]
I like Sunflorians 2!

**Comment by drumdrumdrum3337** [comment]
Sunflorians is lovely! I was also thinking about some new word, that could also lead to a further story. For instance, “farlins” - the legend says that long time ago, everyone living in the land had mastered all the skills - everyone was a farmer and a goblin at the same time. Hence they called themselves farlins - “far” from farmer, and “lin” from goblin. Everyone had a green thumb, knew how to talk to animals, and how to have iron form into different shapes. They spoke the same farlin language. But, the resources started to get scarcer and scarcer, and farlins had to focus more on a certain set of skills, depending on what resources were available to them. The work defined and changed their shapes over time, and even though forced to trade with each other, the relationships between various groups have soured, the language has also developed many dialects. 
..and so on. But yeah, story could sprout pretty much from anything, if it is needed at all, indeed.

**Comment by vincentonecent** [comment]
sun-fam (fam as in family rather than farmers)

**Comment by IntegralDeFluxo** [comment]
I think SUNNIER/SUNNIERS is light, bright, easy, fits easily to humans,
goblins, animals etc, brings a feeling of light and joy, of a happy and
positive day.

Sunniers are "Rastaman vibrations", as Bob Marley would say.


Em sáb, 2 de jul de 2022 22:52, vincentonecent ***@***.***>
escreveu:

**Comment by Tranquilise** [comment]
Son Pharma

**Comment by EJeer** [comment]
**Sunterraneans** or even **Sunflowrraneans** - a native of SunFlower Land! These two names come from a combination of Sunflower Land and terranean (meaning earthly - relating to the earth).

**Reply by esp123** [reply]
Sunterra is quite simple and easy to pronounce

**Comment by stevew00dy** [comment]
So many incredible suggestions here

I've read through them all several times.

My fav is Suntopians

**Reply by thisUsernameAlreadyExists** [reply]
How about we slice it a bit and get `Sutopians` ?

**Comment by esp123** [comment]
Hooman and Goobies

Sun-lings
Sunflower + Saplings

SunBirths
Florians
Settlers
Travelers

**Comment by eArL2384** [comment]
Sunterrarians 🔥

**Comment by ibmisibm** [comment]
Sunflomers (Sunflower + Farmers)

**Comment by veenhdek** [comment]
Pioneers

The whole game experience gives me vibes of living off the land and exploration both of which seem to bring me to pioneer defined as “a person who is among the first to explore or settle a new country or area.”

it is gender and identity neutral and evokes an image to the “medieval” or simplified living vibe of the game

This also will tie nicely into Land Expansion as we pioneer a new Frontier

**Comment by phix06** [comment]
the Suncoms    !!!!!     like sunflower community

**Comment by farsheed98** [comment]
Golkar = floriculturist      for boy
In persian

Sunastasia for girl

And Goldoost ( flower lover ) for goblin

**Comment by adamhannigan** [comment, marked as answer]
Thanks for all of the suggestions everyone. 

For those that suggested "Bumpkins", awesome work. We will be going forward with this name. It has always had deep roots in the project and fits the clumsy/unsophisticated theme of our characters.

Origins of the word come from a similar Dutch word meaning short and stumpy. It is a sign! 🏆

**Comment by On1el** [comment]
If you know the secret of all these names, then users will recognize them and even know them before they see them
Names :
**Suner
**Poto
Pukin
Crot
Cage
Boot (Beet)
Cawer (Caur)
Pip
Rash**
Weth**
